pubsublite/admin.go

```diff
@@ -147,9 +159,36 @@ func (ac *AdminClient) Topics(ctx context.Context, parent string) *TopicIterator
 	}
 }
 
+// CreateSubscriptionOption is an option for AdminClient.CreateSubscription.
+type CreateSubscriptionOption interface {
+	createSubscriptionOption()
+}
+
+type startingOffset struct {
+	backlogLocation BacklogLocation
+}
+
+func (so startingOffset) createSubscriptionOption() {}
+
+// StartingOffset specifies the offset at which a newly created subscription
+// will start receiving messages.
+func StartingOffset(location BacklogLocation) CreateSubscriptionOption {
+	return startingOffset{location}
+}
+
 // CreateSubscription creates a new subscription from the given config. If the
 // subscription already exists an error will be returned.
-func (ac *AdminClient) CreateSubscription(ctx context.Context, config SubscriptionConfig) (*SubscriptionConfig, error) {
+//
+// By default, a new subscription will only receive messages published after
+// the subscription was created. Use StartingOffset to override.
+func (ac *AdminClient) CreateSubscription(ctx context.Context, config SubscriptionConfig, opts ...CreateSubscriptionOption) (*SubscriptionConfig, error) {
+	skipBacklog := true
+	for _, opt := range opts {
+		if _, ok := opt.(startingOffset); ok {
+			skipBacklog = opt.(startingOffset).backlogLocation != Beginning
+		}
```

Comment:
Can avoid the type assertion with Cody's idea above. But, for future type assertions, you can do this:

```suggestion
		if s, ok := opt.(startingOffset); ok {
			skipBacklog = s.backlogLocation != Beginning
		}
```